native/cocos/bindings/auto/jsb_dragonbones_auto.cpp

```diff
@@ -1802,15 +1421,6 @@ bool js_register_dragonBones_EventObject(se::Object* obj) {
     cls->defineFunction("getSlot", _SE(js_dragonBones_EventObject_getSlot)); 
     cls->defineFunction("getAnimationState", _SE(js_dragonBones_EventObject_getAnimationState)); 
     
-    cls->defineStaticProperty("START", _SE(js_dragonBones_EventObject_START_get), _SE(js_dragonBones_EventObject_START_set)); 
-    cls->defineStaticProperty("LOOP_COMPLETE", _SE(js_dragonBones_EventObject_LOOP_COMPLETE_get), _SE(js_dragonBones_EventObject_LOOP_COMPLETE_set)); 
-    cls->defineStaticProperty("COMPLETE", _SE(js_dragonBones_EventObject_COMPLETE_get), _SE(js_dragonBones_EventObject_COMPLETE_set)); 
-    cls->defineStaticProperty("FADE_IN", _SE(js_dragonBones_EventObject_FADE_IN_get), _SE(js_dragonBones_EventObject_FADE_IN_set)); 
-    cls->defineStaticProperty("FADE_IN_COMPLETE", _SE(js_dragonBones_EventObject_FADE_IN_COMPLETE_get), _SE(js_dragonBones_EventObject_FADE_IN_COMPLETE_set)); 
-    cls->defineStaticProperty("FADE_OUT", _SE(js_dragonBones_EventObject_FADE_OUT_get), _SE(js_dragonBones_EventObject_FADE_OUT_set)); 
-    cls->defineStaticProperty("FADE_OUT_COMPLETE", _SE(js_dragonBones_EventObject_FADE_OUT_COMPLETE_get), _SE(js_dragonBones_EventObject_FADE_OUT_COMPLETE_set)); 
-    cls->defineStaticProperty("FRAME_EVENT", _SE(js_dragonBones_EventObject_FRAME_EVENT_get), _SE(js_dragonBones_EventObject_FRAME_EVENT_set)); 
-    cls->defineStaticProperty("SOUND_EVENT", _SE(js_dragonBones_EventObject_SOUND_EVENT_get), _SE(js_dragonBones_EventObject_SOUND_EVENT_set)); 
```

Comment:
Why modify jsb_dragonbones_auto.cpp. I think it has already been updated in the upstream.